src/base/bittorrent/addtorrentparams.h

```diff
@@ -36,13 +36,12 @@
 
 #include "base/path.h"
 #include "base/tagset.h"
+#include "downloadpriority.h"
```

Comment:
You should add this header to .cpp files where you need it and not remove the forward declaration.